examples/NSW_community_gaming/tests/test_nsw_social_housie.catala_en

```diff
@@ -29,8 +29,8 @@ scope Test1:
 ```catala-test-inline
 $ catala Interpret -s Test1
 [ERROR] Syntax error at token "scope"
-Message: expected either 'condition', or 'content' followed by the expected variable type
-Autosuggestion: did you mean "content", or maybe "condition"?
+        Message: expected either 'condition', or 'content' followed by the expected variable type
```

Comment:
Do we want that extra indent ? Also why are we keeping under CI something that has a syntax error?